### PR TITLE
Issue to fix spelling errors in the pricing section and to fix "Buoyant"

### LINKED ISSUE #3592: Spelling errors
#### Description
image
image

#### Page Link:
https://layer5.io/company/news/google-petitions-to-donate-istio-service-mesh-to-cncf
https://layer5.io/pricing

#### Expected Behavior
Correct Bouyant -> Buoyant

The Pricing page contains some spelling errors as well. Kindly fix them.

#### Environment:
- Host OS: MacOS
- Browser: Chrome

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Samipya039**
**Description**

This PR fixes #3592 
The Spelling errors were corrected, and the spelling of the word "Buoyant" was corrected.
**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Follow the steps [here](https://github.com/layer5io/layer5/pull/3595/checks?check_run_id=10492179654) to fix the DCO issue.

And also update the PR title to something meaningful.

**Comment by Samipya039** [comment]
I followed the instructions given there, it would seem that the signoff hasn't changed. What should I do now.

**Comment by Nikhil-Ladha** [comment]
You did execute the commands in sequence, right?
`git rebase HEAD~2 --signoff`
`git push --force-with-lease origin changes`

**Comment by leecalcote** [comment]
A reminder of @Nikhil-Ladha's request, @Samipya039:
> And also update the PR title to something meaningful.

**Comment by Samipya039** [comment]
@Nikhil-Ladha Yes, as per instructiions given in the section, i executed them in order

**Comment by Nikhil-Ladha** [comment]
Well, I don't see signoff added to any of the commits that you have pushed so far.
Try again.

What command are you using to do a commit?

**Comment by l5io** [comment]
🚀 Preview for commit 01fa5a6a0d9b1d55cd2c969a8349ea0c0db7ba57 at: https://63c59609ca0a1c1424527a26--layer5.netlify.app

**Review by eeshaanSA — APPROVED** [review]
Looks good.

**Comment by eeshaanSA** [comment]
"Buoyant" is already fixed, by the way. @Samipya039 . Thanks for the fix on the pricing page. 🚀

**Comment by UtkarshMishra12** [comment]
DCO Failing @Samipya039 Please follow the steps mentioned in the [link](https://github.com/layer5io/layer5/pull/3595/checks?check_run_id=10806341482)

OR

**In your local branch, run**: `git rebase HEAD~3 --signoff`
**Force push your changes to overwrite the branch**: `git push --force-with-lease origin changes`

**Comment by l5io** [comment]
🚀 Preview for commit d03880ca20ee6e1dd3df5800f246fff38892cb75 at: https://63cd4a44c2d9a52c561deb8e--layer5.netlify.app

File: src/sections/Pricing/data.js [hunk 1]
```diff
@@ -46,14 +46,14 @@ export const details = [
     features: [
       {
         feature: "Collaborators for public designs",
-        description: "Invite any Meshery Cloud user, or all Meshery users, to work with you on a public design that control – including making changes and releasing new versions.",
+        description: "Invite any number of Meshery cloud users to work with you on a public design that you can control – including making changes and releasing new versions.",
```
Comment (leecalcote) [inline src/sections/Pricing/data.js:49]: @Samipya039 Meshery Cloud should be capitalized. I'll revert these changes.

**Pull request merged**